src/vaadin-avatar.html

```diff
@@ -42,9 +46,12 @@
       }
     </style>
     <img hidden$="[[!__imgVisible]]" src$="[[img]]" />
-    <svg hidden$="[[__imgVisible]]" id="avatar-icon" viewBox="-50 -50 100 100" preserveAspectRatio="xMidYMid meet">
+    <svg hidden$="[[!__iconVisible]]" id="avatar-icon" viewBox="-50 -50 100 100" preserveAspectRatio="xMidYMid meet">
       <text dy=".35em" text-anchor="middle">&#xea01;</text>
     </svg>
+    <svg hidden$="[[!__abbrVisible]]" id="avatar-abbr" viewBox="-50 -50 100 100" preserveAspectRatio="xMidYMid meet">
+      <text dy=".35em" text-anchor="middle">[[abbr]]</text>
```

Comment:
Any particular reason to use `svg` here and not a `span`? 

We might need to provide parts for styling, not sure if using `svg` would create any obstacles to it.

Reply:
`svg` is use in order to make the `abbr` scalable with the host dimensions changes. Shouldn't cause troubles with styling parts. 🤔